packages/nc-gui/components/project/spreadsheet/components/moreActions.vue

```diff
@@ -268,6 +291,27 @@ export default {
         this.$toast.error(e.message).goAway(3000)
       }
     },
+    async downloadAttachments({ selectedAttachmentField }) {
```

Comment:
We don't do this anymore. We'd like to have a API for downloading attachments. Please check out our latest code and you may check `csvExport` as a reference.